docs/audit-log.rst

```diff
@@ -8,45 +8,58 @@ In the Carbon Black Cloud, *audit logs* are records of various organization-wide
 * Creation of connectors
 * LiveResponse events
 
-The Audit Log API allows these records to be retrieved in JSON format, sorted by time in ascending order
-(oldest records come first). The API call returns only *new* audit log records that have been added since
-the last time the call was made using the same API Key ID. Once records have been returned, they are *cleared*
-and will not be included in future responses.
-
-When reading audit log records using a *new* API key, the queue for reading audit logs will begin three days
-earlier. This may lead to duplicate data if audit log records were previously read with a different API key.
-
-.. note::
-    Future versions of the Carbon Black Cloud and this SDK will support a more flexible API for finding and retrieving
-    audit log records.  This Guide will be rewritten to cover this when it is incorporated into the SDK.
+The Audit Log API allows these records to be retrieved as objects, either by getting the most recent audit logs, or
+through a flexible search API.
 
 API Permissions
 ---------------
 
-To call this API function, use a custom API key created with a role containing the ``READ`` permission on
+To call the Audit Log APIs, use a custom API key created with a role containing the ``READ`` permission on
 ``org.audits``.
 
-Example of API Usage
---------------------
+Retrieving Queued Audit Log Events
+----------------------------------
+
+The Carbon Black Cloud maintains a queue of audit log events for each API key, which is initialized with the last three
+days of audit logs when the API key is created.  This demonstrates how to read audit log events from the queue::
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> from cbc_sdk.platform import AuditLog
+    >>> api = CBCloudAPI(profile='sample')
+    >>> events = AuditLog.get_queued_auditlogs(api)
+    >>> for event in events:
+    ...     print(f"{event.create_time}: {event.actor} {event.description}")
+
+Once audit log events have been retrieved from the queue, they are "cleared" and will not be included in future
+responses to a ``get_queued_auditlogs()`` call.
+
+.. note::
+    Reading queued audit log events using *different* API keys may lead to duplicate data.
+
+Searching for Audit Log Events
+------------------------------
+
+Audit log events may be searched for in a manner similar to other objects within the SDK::
 
-.. code-block:: python
+    # assume "api" contains our CBCloudAPI reference as above
+    >>> query = api.select(AuditLog).where("description:login")
```

Comment:
Instead of "login", use "Logged in" so it matches actual records.
 query = cb.select(AuditLog).where("description:Logged in")

Reply:
Changed in 2 places.